src/bin/js/index_js.mld

```diff
@@ -0,0 +1,6 @@
+{1 Alt-ergo-js}
+
+A Javascript version of the alt-ergo package is available with the command [make js]. The command [make js] will create the javascript file [alt-ergo-js].
+To run this command you will need this aditionnal dependencies [js_of_ocaml-compiler] and [zarith_stubs_js].
+
+The [--timeout] option is unavailable along with the use of input zip file.
```

Comment:
Are the options really unavailable, or are they simply ignored ?

Reply:
Timeout is ignored